pages/1.10/deploying-services/marathon-on-marathon/advanced/index.md

```diff
@@ -1,118 +1,142 @@
 
-   ```bash
-   docker tag mesosphere/marathon-dcos-ee:<mesosphere-tag> <your-repo>/marathon-dcos-ee:<your-tag>
-   ```
-     
-1. Push the Marathon image up to your private Docker registry.
+**Note:** If you are working on a MacOS or Linux machine, you can pre-define all the above variables in your terminal session and just copy and paste the snippets in your terminal:
 
-   ```bash
-   docker push <your-repo>/marathon-dcos-ee:<your-tag>
-   ```
+```
+set -a
+MY_USER_ID="..."
+AGENT_ID="..."
+MESOS_ROLE="..."
+SERVICE_ACCOUNT_ID="..."
+BEGIN_PORT="..."
+END_PORT="..."
+MARATHON_INSTANCE_NAME="..."
+SERVICE_ACCOUNT_SECRET="..."
+DOCKER_REGISTRY_SECRET="..."
+PRIVATE_KEY="..."
+PUBLIC_KEY="..."
+MARATHON_IMAGE="..."
+MARATHON_TAG="..."
+```
+
+
+# Step 1: Preparation
+
+For the following steps, we are assuming that you have already:
+
+1. Pushed the enterprise image of Marathon to your private registry [(Instructions)](/1.10/deploying-services/private-docker-registry#tarball-instructions), under the name `${MARATHON_IMAGE}:${MARATHON_TAG}`.
+1. Stored your private Docker credentials in the secrets store [(Instructions)](/1.10/deploying-services/private-docker-registry/#referencing-private-docker-registry-credentials-in-the-secrets-store-enterprise), under the name `${DOCKER_REGISTRY_SECRET}`.
 
-# Step 2 - Reserve Resources
-In this step, Mesos resources are reserved. Choose the procedure for either [static](#static-reservations) or [dynamic](#dynamic-reservations) reservations.
+    **Warning:** The name of the secret should either be in the root path (ex. `/some-secret-name`) or prefixed with the name of your app (ex. `/${MARATHON_INSTANCE_NAME}/some-secret-name`). Failing to do so, will make root Marathon unable to read the secret value and will fail to launch your custom Marathon-on-Marathon instance.
+
+# Step 2: Reserve Resources (Optional) {.tabs}
+In this step, we are going to define a [Mesos Role](https://mesos.apache.org/documentation/latest/roles/) for our new Marathon instance, and reserve some dedicated resources for it. This step is optional, but it's recommended if you want your new marathon instance to have a minimum resource guarantee for it's tasks.
+
+**Important:** If you choose not to reserve resources, you should use `MESOS_ROLE="*"` in any further occurrence of this variable.
+
+You can choose between [static](#static-reservations) reservations, [dynamic](#dynamic-reservations) reservations or [quotas](#quotas).
+
+| Method | Description |
+|--------|-------------|
+| Static | Dedicates entire agent(s) to the new Marathon |
+| Dynamic | Dedicates a portion of agent(s) resources to the new Marathon |
+| Quotas | Ensures a minimum resource allocation to the new Marathon, regardless of where it's found. |
 
 ## Static Reservations
+A _static_ reservation is achieved by changing the default Mesos Role of a particular agent in order to dedicate all of its resources to the new Marathon instance.
```

Comment:
If we're going to tackle this "reservation part" in another ticket, I'd like to see at least a few sentences which make it clear that:
- This part is optional
- Reasons why it would make sense to do it

Reply:
Hey, just coming up to speed, can you clarify what you mean here?